app/src/main/cpp/skyline/jvm.cpp

```diff
@@ -104,7 +109,41 @@ namespace skyline {
         env->CallVoidMethod(instance, clearVibrationDeviceId, index);
     }
 
+    jobject JvmManager::ShowKeyboard(applet::swkbd::KeyboardConfigVB &config, std::u16string initial_text) {
```

Comment:
Why is `initial_text` in `snake_case` rather than `camelCase`? :stuck_out_tongue: